content/guidelines/reviewing.en.md

```diff
@@ -0,0 +1,109 @@
+7. Verify the version of the document in the contribution tool by clicking on the `verify version` button.
+8. Ensure that all checks generated by the OTA-bot are manually checked.
+9. When you confirm that the term contribution has been made to the correct collection, proceed to add a review.
+10. Merge the contribution.
+
+## How To Review Pull Requests that Update Declarations
+
+When a declaration can no longer be tracked by the Open Terms Archive engine, an issue is created in the collection. This issue contins the details on why the declaration cannot be tracked and the date the challenge was encountered.  
+Contributors are then required to help make this fixes to the declaration in order to maintain the declaration's tracking.
+The updates can be made using the contribution tool and it's effects will be similar to the one seen when adding declaration but with a slight change.
+The pull request created will consist of fewer checks than those that add declarations. The checks will guide the reviewer on the key things to look out for during the review process, and a link to inspect the declaration.
+
+For PRs that update declarations, you should focus should be on two things: history file and the declaration.
+
+- **History file:** The history file is a JSON file that keeps track of a service declaration changes. It contains a `validUntil` property that the date a specific version of a service declaration was last tracked. You have to confirm that this date is the same as the date in the issue opened for the declaration when the bot couldn't track it. This issue is usually included in the PR message. The history file is updated with every `update` PR. You can find more information about the history file [here](https://docs.opentermsarchive.org/contributing-terms/#maintaining-declarations).
+- **Declaration:** for `update` PRs, you only look at the selectors to make sure they are **simple** and also verifiying the **generated declaration** version is relevant and ok.
+
+### Step-by-step Review Guide
+
+1. Click on the inspect the declaration suggestion link to view contribution using the contribution tool.
+2. Check both the significant and insignificant parts of the document.
+    - Ensure that only [simple/basic selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_selectors/Selectors_and_combinators#basic_selectors) are used and avoid complex selectors like [combinators](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_selectors/Selectors_and_combinators#combinators) and [pseudo selectors](https://www.w3schools.com/css/css_pseudo_elements.asp). To simplify, try to avoid selectors with indexes/numbers e.g `:nth-child(n)` and long chains e.g. `div > body > .aside > p > span > a )`
+3. Verify the version of the document in the contribution tool by clicking on the `verify version` button.
```

Comment:
Please include the suggested changes in the above section if you agree with them 🙂

Reply:
Ok. I am putting them in a batch so that I make one commit for all the agreed changes that are similar in what they fix